python/cuml/test/experimental/test_explainer_kernel_shap.py

```diff
@@ -82,7 +82,10 @@ def experimental_test_and_log(cu_shap_values,
 @pytest.mark.parametrize("model", [cuml.LinearRegression,
```

Comment:
How about RandomForest? Can use a smallish n_estimators so the test doesn't take too long.